doc/api/fs.md

```diff
@@ -1074,6 +1075,9 @@ behavior is similar to `cp dir1/ dir2/`.
 <!-- YAML
 added: v22.0.0
 changes:
+  - version: REPLACEME
```

Comment:
Unless I'm missing something this says added in v22.0.0 which would only be around 1 year. The comment says the APIs being promoted have been in place for 3-7 years. Wondering if including the omess added in v22.0.0 was intentiona?

Reply:
@jasnell thanks for confirming, they seem pretty straight forward, so no objections from me in including them, just wanted to make sure it was intentional.